charts/forms-flow-web/templates/deployment.yaml

```diff
@@ -73,10 +73,7 @@ spec:
               key: APPLICATION_NAME
               name: "{{ .Chart.Name }}"
         - name: REACT_APP_USER_ACCESS_PERMISSIONS
-          valueFrom:
-            configMapKeyRef:
-              key: USER_ACCESS_PERMISSIONS
-              name: "{{ .Chart.Name }}"
+          value: "{{ .Values.UserAccesPermissions }}"
```

Comment:
Since this is already coming from config map, shouldn't the config map be changed to use the value from values.yaml and keep the deployment config as is ?

Reply:
Okay I will change it accordingly

Reply:
I have updated the changes @sumesh-aot